addons/packages/cartographer-catalog/0.3.0/cartographer-catalog-README.md

```diff
@@ -0,0 +1,345 @@
+# Out of the Box Supply Chains for TCE
+
+Series of reusable Cartographer Supply Chains and templates for driving
+workloads from source code to running Knative service in a cluster.
+
+## Components
+
+* Cartographer's ClusterSupplyChain and associated resources (templates)
+
+## Configuration
```

Comment:
Can make this a table instead. Can pick the knative or kpack package to see how does the table structure look

Reply:
@kartiklunkad26 I gave the table format a try, but we end up not passing the markdown linting in that way given that we end up pretty much being forced to go over the 80char limit which would end up blocking this PR 🤔  (not really sure why it worked fine for kpack folks, here it complains pretty quickly):

```diff
diff --git a/README.md b/README.md
index 2d71c3f..cf145e6 100644
--- a/README.md
+++ b/README.md
@@ -7,6 +7,19 @@ source code to running Knative service in a cluster.

 * Cartographer's ClusterSupplyChain and associated resources (templates)

+### Configuration
+
+| Value                            | Required/Optional | Description                                                                                                                                                                                                                                        |
+|----------------------------------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `kp_default_repository`          | Optional          | Docker repository used for builder images and dependencies (i.e. `gcr.io/my-project/my-repo` or `mydockerhubusername/my-repo`. Used with the [kp cli](https://github.com/vmware-tanzu/kpack-cli).                                                  |
+| `kp_default_repository_username` | Optional          | Username for `kp_default_repository` (Note: use `_json_key` for GCR)                                                                                                                                                                               |
```

leads to

```
./README.md:12:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:14:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:15:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:16:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:17:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:18:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:19:81 MD013/line-length Line length [Expected: 80; Actual: 301]
./README.md:20:81 MD013/line-length Line length [Expected: 80; Actual: 301]
```

is there a different format we could/should use then? with a list like above we're able to wrap appropriately per the linting rules 🤔